test/openAPI/features/client_update.feature

```diff
@@ -1,16 +1,110 @@
+@method=PUT @endpoint=/client-mgmt/oidc-client/{client_id}
 Feature: API to update existing Open ID Connect (OIDC) client.
 
-  Update OIDC Client Endpoint.
+  @smoke @unit @positive
+  Scenario: The client profile is successfully updated in the Open ID Connect (OIDC) smoke type test
+    Given The user wants to update the client profile in the Open ID Connect (OIDC)
+    And The client profile with "e-health-service" as clientId is created
```

Comment:
as in client_update i think those id should be different for each scenario, the test may fail at this stage because of "duplicate_client_id" error